release_tester/selenium_ui_test/test_suites/analyzers_test_suite.py

```diff
@@ -63,52 +63,52 @@ def test_analyzers(self):
                 
                 if package_version >= semver.VersionInfo.parse("3.9.99"):
                     print('Adding Identity analyzer \n')
-                    analyzers.add_new_analyzer('My_Identity_Analyzer', 0, 128)  # 128 represents required div_id
+                    analyzers.add_new_analyzer('My_Identity_Analyzer', 0, 136)  # 128 represents required div_id
```

Comment:
I think it is possible to come up with universal locators for control elements inside the "Create analyzer" modal window. This way we would not need to pass the div_id parameter here.
For example, "Analyzer name" can be located with this: `//div[label[text()="Analyzer Name"]]/input[not(@disabled)]`
Analyzer type:
`//div[label[text()="Analyzer Type"]]/select[not(@disabled)]`
Frequency
`//div[label[text()="Frequency"]]/input[not(@disabled)]`
etc.

Reply:
I reorganized the whole analyzer's test for UI with new, more stable locators instead of static locators, which was not so intuitive.